docs/src/booklet/6-solvers.md

```diff
@@ -0,0 +1,21 @@
+# QUBO Solvers
+
+## Solvers, Annealers & Samplers
+`ToQUBO`'s main goal is to benefit from non-deterministic samplers, especially *Quantum Adiabatic* devices and other *Annealing* machines. A few `MOI`-compliant interfaces for annealers and samplers are bundled within `ToQUBO` via the `Anneal.jl` submodule and package prototype. Some of them are presented below.
+
+### Simulated Annealing
+Provided by D-Wave's open-source code libraries, this [Simulated Annealing](https://en.wikipedia.org/wiki/Simulated_annealing) engine implements some of the features and configuration you would find using the Quantum API. Its adoption is recommended for basic usage, tests, and during early research steps due to its simplicity and ease of use. It does not implement the most advanced Simulated Annealing algorithm available but performs fairly well on small instances. `Anneal.jl` exports this interface as `SimulatedAnnealer.Optimizer`.
```

Comment:
Provided in the open-source library dwave-neal (link to https://github.com/dwavesystems/dwave-neal), this [Simulated annealing]... and configurations that you would find using the D-Wave Quantum Annealing API.
I would also remove the advice of not using it for research. It is, in fact, a fairy performant implementation of simulated annealing.

Reply:
Also typo: configuration[s]